### PR TITLE
[Application] add README for LLaMA application @open sesame 11/07 16:01

### PR DESCRIPTION
**Pull request by baek2sm**
- remove vocab.json and merges.txt files since each llama version utilizes distinct vocab.json and merges.txt files.
- add README.md file for LLaMA application.

Signed-off-by: Seungbaek Hong <sb92.hong@samsung.com>

### CODE REVIEW

**Review by skykongkong8 — APPROVED** [review]

**Review by DonghakPark — APPROVED** [review]
LGTM! 
if change vocab & merges, they need to changeencoder or tokenizer too 

Resolve
- #2782

**Review by EunjuYang — APPROVED** [review]
LGTM

File: Applications/LLaMA/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- [Paper: The Llama 3 Herd of Models](https://arxiv.org/abs/2407.21783)
+
+## How to run LLaMA
+
+Please refer to the following documents for instructions on setting up and building NNTrainer.
+
+- [Getting Started](../../docs/getting-started.md)
+- [how-to-run-example-android](../../docs/how-to-run-example-android.md)
+
+To run your own LLaMA model, you need to prepare three additional files in application build directory.
```
Comment (djeong20) [inline Applications/LLaMA/README.md:17]: Does "in application build directory" mean inside the jni directory?

**Review by djeong20 — APPROVED** [review]

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by SeoHyungjun — APPROVED** [review]
LGTM!

File: Applications/LLaMA/README.md (hunk 1 above)
Comment (baek2sm) [reply Applications/LLaMA/README.md]: @djeong20 It means the jni directory, but it is not the one where source codes reside.
rather, this directory is generated as a result of building user's application.
I have modified the expression to "inside the application build directory". Thanks!

**Pull request merged**